crates/wit-parser/src/ast/resolve.rs

```diff
@@ -870,7 +923,7 @@ impl<'a> Resolver<'a> {
                 TypeItem::Use(u) => {
                     self.resolve_use(owner, u)?;
                 }
-                TypeItem::Def(_) => {}
+                _ => {}
```

Comment:
Mind going ahead and listing out the other variants? I personally find it best to avoid `_` as a wildcard most of the time since it makes evolution over time a bit harder (more difficult to audit possible locations that `match`)